test/verifiedTests/reconstruction/testModelManipulation/testGenerateRules.m

```diff
@@ -15,28 +15,25 @@
 fileDir = fileparts(which('testGenerateRules'));
 cd(fileDir);
 
-modelsToTry = {'Acidaminococcus_intestini_RyC_MR95.mat', 'Acidaminococcus_sp_D21.mat', 'Recon1.0model.mat', 'Recon2.v04.mat', 'ecoli_core_model.mat', 'modelReg.mat'};
+modelsToTry = {'Recon1.0model.mat', 'ecoli_core_model.mat'};
```

Comment:
Why did you remove these models for testing?

Reply:
Mainly because proper testing (and not just string matching, which doesn't necessarily work, as the formulas can differ in ordering), would take too long if used on all models and using Recon1 + Ecoli should be sufficient. 
In addition: Testing this with a wrong model (i.e. Recon2), doesn't make a lot of sense to me.

Reply:
These models are in there for a reason ... May you put them back? Thanks.